dot/rpc/modules/system_test.go

```diff
@@ -452,8 +478,8 @@ func TestSystemModule_LocalPeerId(t *testing.T) {
 		Multiaddrs: multiAddy,
 	}
 
-	mockNetworkAPI := mocks.NewNetworkAPI(t)
-	mockNetworkAPI.On("NetworkState").Return(ns, nil)
+	mockNetworkAPI := mocks.NewMockNetworkAPI(ctrl)
+	mockNetworkAPI.EXPECT().NetworkState().Return(ns, nil)
```

Comment:
Note the mockery mock was wrong and was returning 2 arguments where it should had returned only one.